newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesListController.php

```diff
@@ -141,17 +144,13 @@ public function getPlaylistsArticlesAction(Request $request, $id)
         }
 
         $playlistArticles = $em->getRepository('Newscoop\Entity\Playlist')
-            ->articles($playlist, null, false, null, null, $onlyPublished, true)->getResult();
+            ->articles($playlist, array(), true, null, null, $onlyPublished, true)->getResult();
 
-        $articlesIds = array();
+        $articles = array();
         foreach ($playlistArticles as $playlistArticle) {
-            $articlesIds[] = $playlistArticle['articleId'];
+            $articles[] = $playlistArticle->getArticle();
```

Comment:
The code in this script newscoop/install/Resources/sql/upgrade/4.4.x/2015.03.19/init_order_in_playlists.php looks exactly the same, check how it calls the method. This also broke because it was returning mixed PlaylistArticles and Articles.